SEED-platform/seed
# Refactor/v3 meters transition

#### Any background context you want to provide?
Part of task of moving to api v3 for tests and FE
#### What's this PR do?
Updates tests and FE for v3 endpoints that were originally part of v2 meters views
#### How should this be manually tested?
`v3/import_files/<id>/pm_meters_preview`: 
- upload example-pm-properties.xlsx (make sure PM ID is set)
- upload example-pm-monthly-meter-usage.xlsx
- you should see a "preview" of the meters to be imported before confirming the import

`v3/import_files/<id>/greenbutton_meters_preview`:
- go to the meters page for a property
- click 'Upload Greenbutton meters', choose example-GreenButton-data.xml
- you should see a "preview" of the meters to be imported before confirming the import

`v3/properties/<id>/meters`:
- upload any kind of meters for a property
- navigate to the meters page of inventory detail and you should see the meter data there.

`v3/properties/<id>/meter_usage`:
- upload any kind of meters for a property
- navigate to the meters page of inventory detail and you should see the meter data there

#### What are the relevant tickets?
#2313
